src/main/kotlin/lotto/view/ResultView.kt

```diff
@@ -3,14 +3,17 @@ package lotto.view
 import lotto.model.LottoRank
 import lotto.model.LottoScore
 import lotto.model.LottoTicket
+import lotto.model.LottoTicket.Companion.automaticSize
+import lotto.model.LottoTicket.Companion.manualSize
 import java.math.BigDecimal
 
 object ResultView {
 
     fun printTickets(tickets: Collection<LottoTicket>) {
-        println("${tickets.size}개를 구매했습니다.")
+        println()
+        println("수동으로 ${tickets.manualSize}, 자동으로 ${tickets.automaticSize}개를 구매했습니다.")
         tickets.forEach { ticket ->
-            println("[${ticket.numbers.map { it.number }.sorted().joinToString(", ")}]")
+            println("[${ticket.lotto.numbers.map { it.number }.sorted().joinToString(", ")}]")
```

Comment:
```suggestion
            println("[${ticket.lotto.numbers.map { it.number }.sorted().joinToString()}]")
```
joinToString의 default separator는 ", " 이므로 생략이 가능합니다. 😄

Reply:
엇 이 부분은 제가 놓친 것 같습니다!! 수정하도록 하겠습니다!!

Reply:
joinToString도 argument name을 활용한다면 아래와 같이 사용할 수 있습니다.
`joinToString(separator = ", ")`